wfe/wfe.go

```diff
@@ -262,6 +262,8 @@ func (wfe *WebFrontEndImpl) relativeEndpoint(request *http.Request, endpoint str
 	return result
 }
 
+var greaseExplanationLink = "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417"
```

Comment:
Also let's rename this along with the other stuff to reflect "random directory entry"